DateTime.d.ts

```diff
@@ -93,6 +93,16 @@ declare namespace ReactDatetimeClass {
          string ('years', 'months', 'days', 'time') as only parameter.
          */
         onViewModeChange?: (viewMode: string) => void;
+        /*
+         Callback trigger when the user navigates to the previous month, year or group of years.
+         The callback receives the amount and type ('month', 'year') as parameters.
+         */
+        onSubtractTime?: (amount: number, type: string) => void;
+        /*
+         Callback trigger when the user navigates to the next month, year or group of years.
+         The callback receives the amount and type ('month', 'year') as parameters.
+         */
+        onAddTime?: (amount: number, type: string) => void;
```

Comment:
These name has `time` in them, but they are triggered for "months, year or group of years". It might be confusing for the user since the component has a time mode.

Naming is hard for these triggers, `onSubtractMonthsOrYears` is more correct but it's long.

What are your thoughts?

Reply:
`onSubtractMonthsOrYears` seems a bit long, maybe something like `navigatePrevious`

Reply:
I like that better, because navigating time doesn't really make sense. Let's go with that, what do you say?